docs/UsersGuide/source/CustomizingTheWorkflow/ConfigWorkflow.rst

```diff
@@ -1277,10 +1280,10 @@ Customized Post Configuration Parameters
 
 Set parameters for customizing the :term:`UPP`.
 
-``USE_CUSTOM_POST_CONFIG_FILE``: (Default: false)
+``USE_CUSTOM_POST_CONFIG_FILE``: (Default: true)
    Flag that determines whether a user-provided custom configuration file should be used for post-processing the model data. If this is set to true, then the workflow will use the custom post-processing (:term:`UPP`) configuration file specified in ``CUSTOM_POST_CONFIG_FP``. Otherwise, a default configuration file provided in the UPP repository will be used. Valid values: ``True`` | ``False``
 
-``CUSTOM_POST_CONFIG_FP``: (Default: "")
+``CUSTOM_POST_CONFIG_FP``: (Default: ``"{{ user.SORCdir }}/ufs-weather-model/tests/parm/postxconfig-NT-fv3lam.txt"``)
```

Comment:
@MichaelLueken Have you created an issue to change this back once we can point to `postxconfig-NT-fv3lam_rrfs.txt`? Just don't want to forget about it!

Reply:
I have an open issue, #916, which talks about the issues with the CRTM on Derecho.  I have gone in and added the temporary fix and that the UPP's `postxconfig-NT-fv3lam_rrfs.txt` file will be used once the CRTM is functioning properly on that machine.